js/plot/static/Plot.svelte

```diff
@@ -192,7 +192,7 @@
 	<div data-testid={"bokeh"} id={divId} class="gradio-bokeh" />
 {:else if type == "altair"}
 	<div data-testid={"altair"} class="altair layout">
-		<Vega {spec} />
+		<Vega {spec} options={{ actions: false }} />
```

Comment:
If we're going to remove this, then I suggest we add a param/prop for it because some users may be relying on the action button being present.

Reply:
Added `show_actions_button` kwarg to configure this.